contracts/src/core/EigenDAServiceManagerStorage.sol

```diff
@@ -29,6 +25,25 @@ abstract contract EigenDAServiceManagerStorage is IEigenDAServiceManager {
      * have to serve after they've deregistered.
      */
     uint32 public constant BLOCK_STALE_MEASURE = 150;
+
+    /** 
+     * @notice The quorum adversary threshold percentages stored as an ordered bytes array 
+     * this is the percentage of the total stake that must be adversarial to consider a blob invalid
+     */
+    bytes public constant quorumAdversaryThresholdPercentages = hex"2121";
```

Comment:
Should these threshold be per quorum?

Reply:
oh nvm, I think this is an array, where index 0 ("21") is for quorum 0, index 1 (the second "21") for quorum 1, etc.